packages/flutter/test/gestures/locking_test.dart

```diff
@@ -33,14 +31,14 @@ class TestGestureFlutterBinding extends BindingBase with GestureBinding {
 
   Future<void> test(VoidCallback callback) {
     assert(callback != null);
-    return _binding.lockEvents(() async {
-      ui.window.onPointerDataPacket(packet);
+    return _binding!.lockEvents(() async {
+      ui.window.onPointerDataPacket?.call(packet);
       callback();
     });
   }
 }
 
-TestGestureFlutterBinding _binding = TestGestureFlutterBinding();
+TestGestureFlutterBinding? _binding;
```

Comment:
It was because the instantiation of `TestGestureFlutterBinding` has the side effect of setting the `GestureBinding.instance` to itself. However, this was down outside the body of the `main` function, so it wasn't being initialized at the right time.  I have reworked this to move its initialization into the main and make it late. Thx.